.github/workflows/main.yml

```diff
@@ -0,0 +1,51 @@
+name: Upload Python Package
+
+# on:
+#   release:
+#     types: [created]
+
+on:
+  push: 
+    branches:
+    - dciborow-patch-1
+    - release/*
+    
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload dist/*      
+    - name: Update version for next release
+      run: |
+        var=$(sed -ne "s/version=['\"]\([^'\"]*\)['\"] *,.*/\1/p" ./setup.py)
```

Comment:
i think we can start with controlling the version manually, then we can add in capabilities for auto-incrementing

Reply:
This should produce a file that can be manually incremented. This would only touch the "patch" number. (these variable names predate shiproom anything by about at least 3 months)

Reply:
or it doesnt work at all.....